ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -6487,3 +6487,19 @@ RFVSC2X:
   - return_fan_speed_percentage_command_2
   implements:
   - OPERATIONAL
+
+DSTLM:
+  description: "Dual section tank with level and temperature monitoring"
+  is_abstract: true
+  uses:
+  - section1_water_temperature_sensor
```

Comment:
we generally use enumerations at the end for identical fields. you would instead have:
`water_temperature_sensor_1`
`water_temperature_sensor_2`
`water_high_level_status_1`
`water_high_level_status_2`
etc.

Reply:
removed thank you